scripts/mcptam_install.bash

```diff
@@ -1,8 +1,25 @@
 #!/bin/bash
+# Normal Use: ./mcptam_install.bash /path/to/catkin_ws 
+# CI Use: ./mcptam_install.bash /path/to/catkin_ws CI
+# Note: You must set up a catkin workspace in advance and either launch this script
+# from the workspace or pass the path to the workspace. Uses catkin_make to build 
+# mcptam.
+
 set -e  # exit on first error
-ROS_VERSION="indigo"
-ROS_BASH="/opt/ros/$ROS_VERSION/setup.bash"
 
+# Determine workspace location
+CATKIN_WS=$1
+CI=false
+if [ $# -eq 2 ] && [ $2 == "CI" ];	then
```

Comment:
So I pretended to be a person who does not read instructions and if you pass no arguments, then the catkin_ws value is not set and this script will error out. Something like the following could be used to catch such problems

if [ $# -eq 0 ] 
  then
    echo "No arguments supplied"
    echo "Please provide the catkin_ws where mcptam will be installed"
    exit 1
fi